packages/openactive-broker-microservice/app.js

```diff
@@ -1147,85 +1147,6 @@ app.post('/assert-unmatched-criteria', function (req, res) {
   }
 });
 
-// /**
```

Comment:
i commented out this stuff in a recent PR when i realised it was obsolete

Reply:
Nice shall we just delete it? It was left in on the assumption the test suite was still using it somewhere

Reply:
Sorry I meant that I commented it out in a recent PR and that this PR deletes it

Reply:
Ha! So you did - I didn't see the red text background for some reason...